### DISCUSSION TITLE
Support ed25519 public keys in certificate

### DISCUSSION THREAD

**Discussion by saijannali** [MasterKale/SimpleWebAuthn#645] [Ideas]
Hi,

I'm a newbie to this space.

There is an error when attempting to use a certificate containing a ed25519 (-8) public key in certificate for packed attestation.

{"error": "Certificate public key contained unexpected algorithm ID 1.3.101.112"}

It would be cool to support. For reference: https://github.com/MasterKale/SimpleWebAuthn/blob/master/packages/server/src/helpers/convertX509PublicKeyToCOSE.ts

If I am missing anything, please let me know.

**Comment by MasterKale** [comment]
Heya @saijannali I don't have problems with supporting these attestation statement certs, I've just never run into them before? If you're able to, can you create an issue about this including the attestation statement and root cert needed to verify the full attestation? I can develop against it to add Ed25519 cert support.
